$NetBSD: patch-configure,v 1.2 2014/08/21 21:41:52 wiz Exp $

First chunk: ./configure: 28988: Syntax error: Bad substitution
Others: test(1) portability fix.

--- configure.orig	2014-08-16 16:06:45.000000000 +0000
+++ configure
@@ -28985,7 +28985,6 @@ fi
 have_fpx='no'
 FPX_LIBS=''
 SAVED_LIBS=${LIBS}
-LIBS=${LIBS/-ldjvulibre/}
 if test "$with_fpx" != 'no'; then
     { $as_echo "$as_me:${as_lineno-$LINENO}: result: -------------------------------------------------------------" >&5
 $as_echo "-------------------------------------------------------------" >&6; }
@@ -33879,7 +33878,7 @@ $as_echo "no" >&6; }
 fi
 
 
-if test "$MPEGDecodeDelegate" == "$MPEGDecodeDelegateDefault" ; then
+if test "$MPEGDecodeDelegate" = "$MPEGDecodeDelegateDefault" ; then
   MPEGDecodeDelegateDefault='ffmpeg'
   # Extract the first word of ""$MPEGDecodeDelegateDefault"", so it can be a program name with args.
 set dummy "$MPEGDecodeDelegateDefault"; ac_word=$2
@@ -33964,7 +33963,7 @@ $as_echo "no" >&6; }
 fi
 
 
-if test "$MPEGEncodeDelegate" == "$MPEGEncodeDelegateDefault" ; then
+if test "$MPEGEncodeDelegate" = "$MPEGEncodeDelegateDefault" ; then
   MPEGEncodeDelegateDefault='ffmpeg'
   # Extract the first word of ""$MPEGEncodeDelegateDefault"", so it can be a program name with args.
 set dummy "$MPEGEncodeDelegateDefault"; ac_word=$2
